frontend/src/components/Jarvis/index.jsx

```diff
@@ -6,7 +6,7 @@ import { useState, useEffect } from "react";
 import axios from "axios";
 import SJarvis from "./style";
 import Modal from "./Modal";
-import Cards from "../Cards";
+import Card from "../Cards";
```

Comment:
That's a terrible idea: you should rename the file to match it's component name instead